### PR TITLE
adds condition of testing

### PR DESCRIPTION
**Pull request by capetillo**
## TEST: Format Time utils

Added tests to test the functions of `formatTime` utils functions and .yaml file to run github actions

### CODE REVIEW

File: src/tests/unit/utils/formatTime.test.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe('formatUtils.js', () => {
 
     it('returns an empty string for an invalid date', () => {
```
Comment (mgdaily) [inline src/tests/unit/utils/formatTime.test.js:11]: Is this an invalid date, still? It appears to be valid?

File: src/tests/unit/utils/formatTime.test.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ describe('formatUtils.js', () => {
 
     it('returns an empty string for an invalid time', () => {
       const time = 'Wed Aug 21 2024 16:15:00 GMT-0700 (Pacific Daylight Time)'
```
Comment (mgdaily) [inline src/tests/unit/utils/formatTime.test.js:24]: Also appears to be valid?

**Review by mgdaily — COMMENTED** [review]
Just a lil confused about tests that are claiming to be for invalid dates but are returning valid date strings

File: src/tests/unit/utils/formatTime.test.js (hunk 1 above)
Comment (capetillo) [reply src/tests/unit/utils/formatTime.test.js]: forgot to update description! thanks Matt hehe

**Review by mgdaily — APPROVED** [review]
Nice work!

**Pull request merged**